src/terra-dev-site/IssueForm/CommonForm.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Field } from 'react-final-form';
+import TextareaField from 'terra-form-textarea/lib/TextareaField';
+import { propTypes, validateForm } from './Helper';
+
+const FormTitle = (props) => {
```

Comment:
Lets break each of these components into their own file

Reply:
Changed: [5eff56a](https://github.com/cerner/terra-ui/pull/151/commits/5eff56a5761ea4b8acbfe97a933e639186f91e48)